docs/devctr-image.md

```diff
@@ -46,9 +46,31 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 
 1. Build a new container image with the updated Dockerfile.
 
+   The below script is used to build a docker image.
```

Comment:
This line is a little redundant. Maybe replace it with `Run the following command:`

Reply:
changed